File: tools/licenses/test_utils.py
FASTVC-BENCH-CORPUS:chromium-main-v1-943b94ae-1c74-4335-94fa-ceb4d277cea8
# Copyright 2022 The Chromium Authors
# Use of this source code is governed by a BSD-style license that can be
# found in the LICENSE file.
"""Helper functions for tests in //tools/licenses."""
import os

_OS_ROOT = os.path.abspath(os.sep)


def path_from_root(*paths: str):
  return os.path.join(_OS_ROOT, *paths)

File: chrome/test/data/webui/chromeos/settings/os_apps_page/app_parental_controls_page/test_utils.ts
FASTVC-BENCH-CORPUS:chromium-main-v1-943b94ae-1c74-4335-94fa-ceb4d277cea8
// Copyright 2024 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

import type {appParentalControlsHandlerMojom} from 'chrome://os-settings/os_settings.js';

/**
 * Creates an app for testing purpose.
 */
export function createApp(id: string, title: string, isBlocked: boolean):
    appParentalControlsHandlerMojom.App {
  return {id, title, isBlocked};
}
